### ISSUE TITLE
Performance issues during xlsx-template loading

### ISSUE THREAD

**Issue by ivanfemia** [abap2xlsx/abap2xlsx#197]
Considerable performance fall during template loading. Found that the problem occurs with XLSX-files which have last COL element with 'MAX' attribute set to 16384 - i.e., maximum number of columns (see attachment). Path in XML (see the last COL-element): 

<pre><code>[xl]
    [worksheets]
        [sheet1] -> [worksheet]
            [cols]
                [col]
...
                [col]-min = 64, [col]-max = 16384, [col]-width = 9.140625, [col]-style = 36
</code></pre>

 As I understand, last column's attributes are set to default and thus should normally not be present.

At first, I tried to clear formatting of columns outside the template but the file still had these attributes. So, during template loading ZCL_EXCEL_READER_2007 creates column dimensions with default formatting for 16K objects which is, 1) unnecessary, 2) resource consuming, 3) performance decreasing.

I temporary solved the issue by declaring MAX_COL constant set to 16384 and comparing MAX attribute to it in the LOAD_WORKSHEET method, lines after 160:

<pre><code>WHILE column_elem IS BOUND.
  fill_struct_from_attributes( EXPORTING ip_element = column_elem CHANGING cp_structure = column ).
  column_elem ?= iterator->get_next( ).
  IF column-max = max_col.
    EXIT.
  ENDIF.</code></pre>

**Comment by ivanfemia** [comment]
Sorry for confusing. Columns 64 to MAX were formatted with Trebuchet MS font. I guess the person responsible for the template just selected all and set new font for the selection. Therefore, the easiest way to solve the problem is to format a template correctly, i.e., only used columns. However, this situation still can occur, so it has to be dealt with.

**Comment by ivanfemia** [comment]
Invalidate the issue right now.

**Comment by ivanfemia** [comment]
I am very sorry, but right now I have no ability to check whether the issue was resolved or not. Last time I had to patch the loader to use "span" attribute of a "row" element (for instance, "1:68"). I took a maximum right value of all spans (rows loop)  and compared current column number to it (columns loop). If it exceeded a MAX_SPAN value, the loop was terminated (see example coding below).

<pre><code>
WHILE column_elem IS BOUND.
  fill_struct_from_attributes( EXPORTING ip_element = column_elem CHANGING cp_structure = column ).
  column_elem ?= iterator->get_next( ).
...
  col = column-min.
  WHILE col LE column-max AND col LE max_span.
    ...
  ENDWHILE.</code></pre>
